### PR TITLE
Improve PolicyStudioWrapper and migrate platform policies screen

### PR DESCRIPTION
**Pull request by gaetanmaisse**
**Issue**

https://github.com/gravitee-io/issues/issues/6370

**Description**

- Improve Angular PolicyStudioWrapper by fixing inputs, adding missing outputs and behaviours
- Create some Angular services
- Migrate platform policies screen to angular and to the new PolicyStudioWrapper

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-ylshssrlrk.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6370-complete-policy-studio-wrapper/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-wpzcsrsmll.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6370-complete-policy-studio-wrapper/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

File: gravitee-apim-console-webui/src/shared/components/gio-policy-studio-wrapper/gio-policy-studio-wrapper.component.ts [hunk 1]
```diff
@@ -52,7 +54,28 @@ export class GioPolicyStudioWrapperComponent implements OnInit {
   sortable: boolean;
 
   @Input()
-  policies: unknown[];
+  set policies(policies: PolicyListItem[]) {
+    this._policies = policies
+      .map((policy) => (!isNil(policy.category) ? policy : { ...policy, category: this.unknownPolicyCategory }))
+      .sort((policyA, policyB) => {
```
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/shared/components/gio-policy-studio-wrapper/gio-policy-studio-wrapper.component.ts]: maybe we could use lodash `sortBy` 🤸‍♂️

**Review by ThibaudAV — APPROVED** [review]

File: gravitee-apim-console-webui/src/shared/components/gio-policy-studio-wrapper/gio-policy-studio-wrapper.component.ts (hunk 1 above)
Comment (gaetanmaisse) [reply gravitee-apim-console-webui/src/shared/components/gio-policy-studio-wrapper/gio-policy-studio-wrapper.component.ts]: 🏋🏻

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-obuwvlbetu.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6370-complete-policy-studio-wrapper/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Pull request merged**